tests/conftest.py

```diff
@@ -28,6 +29,16 @@ def pytest_configure(config):
     )
 
 
+def get_tests_daft_runner_name() -> Literal["ray"] | Literal["py"] | Literal["native"]:
+    """Test utility that checks the environment variable for the runner that is being used for the test"""
+    name = os.getenv("DAFT_RUNNER")
+    assert name is not None, "Tests must be run with $DAFT_RUNNER env var"
```

Comment:
if running locally we may not have the env var set, and this will error. Use h`_get_runner_config_from_env()` function in daft.context instead? 

I don't think it has any side effects.

Reply:
Maybe we should just force developers to set `DAFT_RUNNER` explicitly locally? Might encourage folks to test other backends explicitly too